### DISCUSSION TITLE
What's a "container"?

### DISCUSSION THREAD

**Discussion by DanMan** [pyload/pyload#4164] [Help] [answered]
When you add a new "package" (again, wording here is confusing), there's a file upload to "Upload a container". I'm at a loss what that's supposed to be and I haven't found any documentation on it.

So what sort of "container" can you upload there? A torrent maybe? Some file with a list of URLs in it?

**Comment by GammaC0de** [comment, marked as answer]
A container can be a text file that contains a list of URLs, a torrent file of a [DLC file](https://fileinfo.com/extension/dlc) that some sites provide.
